source/help/getting-started/light-install.rst

```diff
@@ -0,0 +1,55 @@
+Light Install Guide
+===================
+
+Mattermost is a messaging and collaboration platform. With Mattermost, you can integrate the tools you use every day into one place and never miss a notification or task. 
+
+Before you get started you'll need:
+
+* A clean Ubuntu server (18.04 or 20.04) with root level access
+* (Recommended) A domain name pointing to your server
+* (Recommended) Email addresses of your team members so you can invite them to the server
+
+Installation
+-------------
+
+Install Mattermost using `Mattermost Omnibus <https://docs.mattermost.com/install/mattermost-omnibus.html>`_.
+
+Open a terminal window and enter: 
+
+.. code-block::
+
+  curl -o- deb.packages.mattermost.com/repo-setup.sh | sudo bash
+
+Note: Although the recommended way to install and configure Omnibus is with SSL enabled, if you want to use or test without it, you can run: 
```

Comment:
Do we want to render this as a note with the `.. note::` RST formatting?

Reply:
Good catch, thanks! It's been updated.